### DISCUSSION TITLE
bypassing Nginx URL Signing

### DISCUSSION THREAD

**Discussion by Shadab-Airtel** [Sunbird-Ed/Community#42] [Sunbird ED configuration discussion]
hi, as per our understanding of DIKSHA architecture outgoing traffic passes through Nginx Gateway. Please confirm if we can bypsass Nginx gateway using URL signing.

**Comment by sharathkashyap** [comment]
@Shadab-Airtel Apologies for delayed response. 
@keshavprasadms Will you be able to help on this

**Reply by beepdot** [reply]
Could you please provide some more additional information on URL signing and how are you planning to implement it? Maybe a simple example or usecase will help us in providing you an answer.

**Comment by Shadab-Airtel** [comment]
Rikin/Amit: please help articulate use cases as requested by sunbird team.

Regards,
Mohd Shadab
9911120419

**Reply by sharathkashyap** [reply]
@Shadab-Airtel Could you please help us with inputs as requested by @keshavprasadms

**Comment by Shadab-Airtel** [comment]
Rikin, awaiting inputs.

Regards,
Mohd Shadab
9911120419

From: Mohd. Shadab
Sent: 18 July 2022 15:27
To: 'Amit Kaushik' ***@***.***>; Rikin Shah ***@***.***>
Cc: Mention ***@***.***>; 'Sunbird-Ed/Community' ***@***.***>; Sunbird-Ed/Community ***@***.***>
Subject: RE: [Sunbird-Ed/Community] bypassing Nginx URL Signing (Discussion #42)

Rikin/Amit: please help articulate use cases as requested by sunbird team.

Regards,
Mohd Shadab
9911120419

**Comment by sharathkashyap** [comment]
Locking this thread due to inactivity.
